srcpkgs/xcolor/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'xcolor'
+pkgname=xcolor
+version=0.5.0
+revision=1
+archs="x86_64"
```

Comment:
no

Reply:
what is the issue?

Reply:
What's the issue with other architectures that made you restrict the package only to glibc x86_64?

Reply:
The [PKGBUILD](https://aur.archlinux.org/packages/xcolor) for archlinux only mentions x86_64. I do admit I didn't look that deep into it, since I don't have a 32-bit machine